recipes/fbclient/3.x.x/conanfile.py

```diff
@@ -0,0 +1,89 @@
+import functools
+import os
+
+from conans import ConanFile, CMake, tools
+
+required_conan_version = ">=1.43.0"
+
+
+class FbClientConan(ConanFile):
+    name = "fbclient"
+    description = (
+        "Firebird is a relational database offering many ANSI SQL standard "
+        "features that runs on Linux, Windows, MacOS and a variety of Unix "
+        "platforms."
+    )
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.firebirdsql.org/"
+    topics = "sql", "database", "db"
+    license = "LicenseRef-IDPL.txt"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+    generators = "cmake", "cmake_find_package_multi"
+    # NOTE: The cpp files are from epp files pre-processed using the gpre tool
+    #       See https://github.com/conan-io/conan-center-index/issues/9307
+    exports_sources = "CMakeLists.txt", "array.cpp", "blob.cpp"
```

Comment:
@uilianries The files can be found here https://github.com/FirebirdSQL/firebird/tree/v3.0.8/src/yvalve

Reply:
@friendlyanon Thank you for the clarification. In this case, you can download them directly, don't need to copy. As real example, you may check ApprovalTests conandata: https://github.com/conan-io/conan-center-index/blob/master/recipes/approvaltests.cpp/all/conandata.yml